<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FtpUploader</class>
 <widget class="QWidget" name="FtpUploader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="hostLabel">
     <property name="text">
      <string>Host</string>
     </property>
     <property name="buddy">
      <cstring>hostEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="hostEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="portLabel">
     <property name="text">
      <string>Port</string>
     </property>
     <property name="buddy">
      <cstring>portSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="portSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="loginLabel">
     <property name="text">
      <string>Login</string>
     </property>
     <property name="buddy">
      <cstring>loginEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="loginEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="passworLabel">
     <property name="text">
      <string>Hasło</string>
     </property>
     <property name="buddy">
      <cstring>passwordEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="passwordEdit">
     <property name="toolTip">
      <string>Zapisywane w konfiguracji hasło nie jest czystym tekstem,  jednak jego kodowanie nie jest silne.
Jeśli nie chcesz zapisywać hasła pozostaw to pole puste, program zapyta o nie przy wysyłaniu fororelacji.</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="urlLabel">
     <property name="text">
      <string>Adres do katalogu głównego</string>
     </property>
     <property name="buddy">
      <cstring>urlEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="urlEdit">
     <property name="toolTip">
      <string>Adres do katalogu głównego ftp jest częścią linku tworzonego dla każdego zdjęcia.
Źle wpisany spowoduje, że wszystkie adresy do zdjęć będą niewłaściwe.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>hostEdit</tabstop>
  <tabstop>portSpinBox</tabstop>
  <tabstop>loginEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>urlEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
